tests/test_adroll_full_replication.py

```diff
@@ -111,38 +155,76 @@ def test_run(self):
         # first_sync_records["ads"] == second["ads"]
         for stream in full_streams:
             with self.subTest(stream=stream):
-
+                # RECORD COUNT
+                record_count_1 = first_sync_record_count.get(stream, 0)
+                record_count_2 = second_sync_record_count.get(stream, 0)
+                # ACTUAL RECORDS
+                records_from_sync_1 = set(row.get('data').get('eid')
+                                          for row in first_sync_records.get(stream, []).get('messages', []))
+                records_from_sync_2 = set(row.get('data').get('eid')
```

Comment:
These two should also be `row.get('data', {}).get('eid')` if safety is the goal.